<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WheelWidget</class>
 <widget class="QWidget" name="WheelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="numberOfTeethLabel">
       <property name="text">
        <string>Number of teeth</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="numberOfTeethSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="externalDiameterLabel">
       <property name="text">
        <string>External diameter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="externalDiameterDoubleSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string> mm</string>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="primitiveDiameterLabel">
       <property name="text">
        <string>Primitive diameter</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="primitiveDiameterDoubleSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="suffix">
        <string> mm</string>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="stackedLabel">
       <property name="text">
        <string>Stacked</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="stackedCheckBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="holeDiameterLabel">
       <property name="text">
        <string>Hole diameter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="holeDiameterDoubleSpinBox">
       <property name="suffix">
        <string> mm</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="numberOfLighteningHolesLabel">
       <property name="text">
        <string>Number of
Lightening holes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="numberOfLighteningHolesSpinBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="armWidthLabel">
       <property name="text">
        <string>Arm width</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="armWidthDoubleSpinBox">
       <property name="suffix">
        <string> mm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="addPushButton">
         <property name="text">
          <string>Add </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addStackedPushButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Add Stacked</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="deletePushButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportPushButton">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
